build/charts/theia/provisioning/datasources/create_table.sh

```diff
@@ -182,8 +207,13 @@ clickhouse client -n -h 127.0.0.1 <<-EOSQL
         sourcePodNamespace,
         destinationPodNamespace;
 
+    {{- if .Values.clickhouse.cluster.enable }}
```

Comment:
Looking at the way the DB is created we cannot avoid destroying/recreating the database when switching to clustered mode. (in theory we could have some complex logic for renaming old tables, creating new tables, and moving data around).

I think a strategy here could be to document that switching is not supported. Even if one toggled cluster.enable from False to True (or vice versa), we'd need to delete old tables to make sure the DB is properly created, which will work only for in-memory DB.
The other part of the strategy would to default to clustered mode, and possibly also deprecate non-clustered mode. At the end of the day, for a small devtest env it is enough to run a single CH replica, and a single ZK instance - is that correct?

Reply:
It sounds good to add a devtest env first. Deploying single CH replica is already supported. I can add an option to deploy single ZK instance and doc this. But I'm not sure whether we should deprecate non-clustered mode, as it requires less resource by not deploying ZK, which might be useful for light-weighted env.

Reply:
Updated to allow any number of ZK replica, and also the docs. Thanks!